### ISSUE TITLE
Add support for ORIGIN messages

### ISSUE THREAD

**Issue by agentultra** [agentultra/postgresql-replicant#21]
Handle ORIGIN messages so we know where changes came from upstream in a cluster setting

https://www.postgresql.org/docs/14/protocol-logicalrep-message-formats.html

`wal2json` supports adding this data with a config option.

**Comment by agentultra** [comment]
This would be nice to have, waiting on wal2json to see if it will support these

**Comment by agentultra** [comment]
We need an example of what this looks like in the `wal2json` change messages
